auth0/v3/authentication/authorize_client.py

```diff
@@ -9,9 +9,6 @@ class AuthorizeClient(AuthenticationBase):
         domain (str): Your auth0 domain (e.g: username.auth0.com)
     """
 
-    def __init__(self, domain):
```

Comment:
this is a breaking change. Why have you removed this? (my comment applies to all modified files)

Reply:
I removed this since all of AuthenticationBase's subclasses had identical `__init__` methods, and it seemed cleaner to let them all use the base class's new implementation as opposed to adding parameters and `super` calls throughout all these.

I think since they will fall back to use `AuthenticationBase.__init__`, and it accepts the domain arg in the same position, that it shouldn't break anything.